src/components/modules/moim-detail/MoimDetailSummary.tsx

```
@@ -178,13 +162,13 @@ const MoimDetailSummary = ({ moim }: MoimDetailSummary) => {
                 disabled={isCanCelMoimPending}
-                className="h-full w-1/2 rounded-[12px] border border-gray-100 text-sm font-medium text-gray-500 sm:text-base md:text-xl"
+                className="h-full w-1/2 rounded-[12px] border border-gray-100 text-sm font-medium text-gray-500 hover:cursor-pointer sm:text-base md:text-xl"
               >
-                {isCanCelMoimPending ? "취소중..." : "취소하기"}
+                {isCanCelMoimPending ? "취소중..." : "모임 취소하기"}
               </button>
-              <DropdownMenu>
+              <DropdownMenu modal={false}>
                 <DropdownMenuTrigger asChild>
-                  <button className="w-1/2 rounded-[12px] bg-green-500 text-sm font-bold text-white sm:h-12 sm:text-base md:h-15 md:text-xl md:font-semibold">
+                  <button className="w-1/2 rounded-[12px] bg-green-500 text-sm font-bold text-white hover:cursor-pointer sm:h-12 sm:text-base md:h-15 md:text-xl md:font-semibold">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)